reference/Client-API.yaml

```diff
@@ -1725,12 +1725,7 @@ paths:
                   signal: stop
               properties:
                 signal:
-                  type: string

```

Comment:
Why was this changed?

Reply:
Didn't know Stoplight hasn't reverted that